### ISSUE TITLE
Autocomplete tutors field

### ISSUE THREAD

**Issue by ogoletti** [UCL-INGI/INGInious#711]
The field to add tutors in `admin/COURSE_ID/settings` sometimes behaves strangely. For some tutors, the autocomplete feature does not find them somehow (based on firstname or lastname)... If I ask the tutor for her username, then it will find it easily (and it then works for both firstname and lastname). 
I understand this might be difficult to reproduce I can help more on demand.

**Comment by Drumor** [comment]
Some weird behavior but in the end, all users are found.
I close this one for now, if need rework, we can still open it again.

**Comment by ogoletti** [comment]
It seems the field is case-sensitive. 
Even though I think I observed weird behaviour. Like a name not matching because of case-sensitiveness, then matching with the same keyword after I found it with the ID... Maybe I did something wrong. I'll repost if I encounter it again (next year :D). 

@GuillaumeDerval asked me to reopen with this link https://docs.mongodb.com/manual/reference/operator/query/regex/#perform-case-insensitive-regular-expression-match as we discussed whether case-insensitive would be better for this field. 

In any case, if the field is case-sensitive it would be nice to have a warning or so on the page.
